examples/src/main/java/io/opencensus/examples/trace/MultiSpansScopedTracing.java

```diff
@@ -55,9 +59,27 @@ private static void doWork() {
    * @param args the main arguments.
    */
   public static void main(String[] args) {
+
+    // WARNING: Be careful before you set sampler value to always sample, especially in
+    // production environment. Trace data is often very large in size and is expensive to
+    // collect. This is why rather than collecting traces for every request(i.e. alwaysSample),
+    // downsampling is prefered.
+    //
+    // By default, OpenCensus provides a probabilistic sampler that will trace once in every
+    // 10,000 requests. If you prefer to use probabilistic sampler, you might not see trace data
```

Comment:
s/". If you prefer to use probabilistic sampler"/", that's why if default probabilistic sampler is used here"

and everywhere where you have this comment.